includes/Html.php

```diff
@@ -627,15 +627,20 @@ public static function inlineStyle( $contents, $media = 'all' ) {
 	 *
 	 * @param $url string
 	 * @param $media mixed A media type string, like 'screen'
+	 * @param $crossorigin Configure the CORS requests for the fetched data
 	 * @return string Raw HTML
 	 */
-	public static function linkedStyle( $url, $media = 'all' ) {
-		return self::element( 'link', array(
+	public static function linkedStyle( $url, $media = 'all', $crossorigin = null ) {
```

Comment:
This is core MW file. We want to avoid as much as possible changing it. What do you think of creating `WikiaHtml` class which extends the possibility of this method and using it in `WikiaSkin` part?